frontend/pages/policies/ManagePoliciesPage/components/CalendarEventsModal/CalendarEventsModal.tsx

```diff
@@ -104,11 +119,19 @@ const CalendarEventsModal = ({
       });
       const newFormData = { ...formData, policies: newFormPolicies };
       setFormData(newFormData);
-      setFormErrors(validateCalendarEventsFormData(newFormData));
     },
     [formData]
   );
 
+  const onUpdatePolicyEnabledCalendarEvents = () => {
+    const errors = validateForm(formData);
```

Comment:
related to above, I don't think `validate` should both set error state and return the same errors